libbpf-cargo/src/build.rs

```diff
@@ -111,26 +111,38 @@ fn check_clang(debug: bool, clang: &Path, skip_version_checks: bool) -> Result<(
 ///
 /// for each prog.
 fn compile_one(debug: bool, source: &Path, out: &Path, clang: &Path, options: &str) -> Result<()> {
-    let arch = match std::env::consts::ARCH {
-        "x86_64" => "x86",
-        "aarch64" => "arm64",
-        _ => std::env::consts::ARCH,
-    };
-
     if debug {
         println!("Building {}", source.display());
     }
 
     let mut cmd = Command::new(clang.as_os_str());
+    let mut arch_set = false;
+
+    /*
```

Comment:
nit: in Rust, i think the convention is to use "//" for comments. do you mind changing this to using "//"?

Reply:
Sure, I updated the commit to use change "/*" to "//", thank you.